lib/awful/layout/init.lua.in

```diff
@@ -109,7 +109,15 @@ function layout.arrange(screen)
     p.geometry = capi.screen[screen].geometry
     p.clients = client.tiled(screen)
     p.screen = screen
-    layout.get(screen).arrange(p)
+
+    -- Apply layout geometry
+    local gs = layout.get(screen).arrange(p)
+    for k, gp in ipairs(gs) do
+        gp[2].height = gp[2].height - gp[1].border_width * 2
+        gp[2].width = gp[2].width - gp[1].border_width * 2
+        gp[1]:geometry(gp[2])
+    end
+
```

Comment:
I do like this idea. It makes much more sense than throwing them away each time. However this is my first time digging into awesome's source, so I don't really know where to get started. Any tips on where I can look to get an idea?